SPECS/clamav/clamav.spec

```diff
@@ -1,26 +1,24 @@
 %{!?python2_sitelib: %global python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 %{!?python3_sitelib: %global python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-
 Summary:        Open source antivirus engine
 Name:           clamav
-Version:        0.103.0
+Version:        0.103.2
 Release:        1%{?dist}
-License:        ASL 2.0 and BSD and bzip2-1.0.4 and GPLv2 and LGPLv2+ and MIT and Public Domain and UnRar
-Group:          System Environment/Security
+License:        ASL 2.0 AND BSD AND bzip2-1.0.4 AND GPLv2 AND LGPLv2+ AND MIT AND Public Domain AND UnRar
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Security
```

Comment:
I suggest removing the group. This is an obsolete feature which does not add any value.

Reply:
Going to go ahead and leave it for now- guidance last time I checked (around October) was that this may eventually be something we want to use.